website/content/en/docs/concepts/scheduling.md

```diff
@@ -191,6 +191,14 @@ Then the pod can declare that custom label.
 
 See [nodeSelector](https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#nodeselector) in the Kubernetes documentation for details.
 
+## Preferences
+
+Karpenter is aware of preferences (node affinity, pod affinity, pod antiaffinity, and pod topology) and treats them as requirements in almost all circumstances. As a result, Karpenter also treats preferred affinities as required affinities when constructing requirements for a pod. When these requirements can not be met, preferences are relaxed where possible and the remaining requirements are tried again.
```

Comment:
```suggestion
Karpenter is aware of preferences (node affinity, pod affinity, pod antiaffinity, and pod topology) and treats them as requirements in almost all circumstances. As a result, Karpenter also starts by treating preferred affinities as required affinities when constructing requirements for a pod. When these requirements can not be met, preferences are relaxed one-at-a-time by ascending weight (lowest weight is relaxed first) and the remaining requirements are tried again.
```

Small wording change, but this just implies that it starts in one way and then relaxes them over time